pom.xml

```diff
@@ -136,11 +136,6 @@
 							<filtering>true</filtering>
 							<targetPath>WEB-INF</targetPath>
 						</resource>
```

Comment:
I think this entry in the pom put the properties file on the java class path.  Have you confirmed it's not needed?

Reply:
no i have not but I see your point. i have restored this in the pom